hummingbot/client/command/rate_command.py

```diff
@@ -39,7 +38,7 @@ async def show_rate(self,  # type: HummingbotApplication
     @staticmethod
     async def oracle_rate_msg(pair: str,
                               ):
-        pair = pair.upper()
+        pair = eval(pair.upper())
```

Comment:
It does not make sense at all to evaluate a trading pair as if it were a function.

Reply:
This seemed like a common technique to remove extra single and double quotes from string. Would you suggest to use json instead. This may not be something you guys want to 'fix' to beginning with